check_sites.py

```diff
@@ -36,9 +36,12 @@ def find_diff_sets(old_sets, new_sets):
              for primary, fps in new_sets.items()
              if fps != old_sets.get(primary)
             }
-
     subtracted_sets = {primary: old_sets[primary] for primary in set(old_sets) - set(new_sets)}
-
+    # Make sure that subtracted sets aren't just sets with changed primaries
+    for _, fps in diff_sets.items():
+        for primary in subtracted_sets:
+            if fps.includes(primary):
+                subtracted_sets -= primary
```

Comment:
I think a simpler way of handling this scenario would be to include it in the initial definition of `subtracted_sets` like this:

```suggestion
    subtracted_sets = {
        primary: old_sets[primary]
        for primary in set(old_sets) - set(new_sets)
        if primary not in new_sets
    }
```

Reply:
Oh, I guess we'd have to make the guard call `fps.includes`:

```suggestion
    subtracted_sets = {
        primary: old_sets[primary]
        for primary in set(old_sets) - set(new_sets)
        if not any(fps.includes(primary) for fps in new_sets)
    }
```

Reply:
Done